be/src/runtime/local_tablets_channel.cpp

```diff
@@ -445,6 +445,9 @@ void LocalTabletsChannel::add_chunk(Chunk* chunk, const PTabletWriterAddChunkReq
 }
 
 void LocalTabletsChannel::_flush_stale_memtables() {
+    if (_immutable_partition_ids.empty() && config::stale_memtable_flush_time_sec <= 0) {
```

Comment:
Why check `_immutable_partition_ids.empty()` ?

Reply:
this is for automatic bucket